lighthouse-core/report/html/report-styles.css

```diff
@@ -1246,7 +1281,6 @@
 }
 
 .lh-category-header .lh-score__gauge {
-  max-width: 400px;
```

Comment:
just unnecessary or do we want the gauges to get bigger than this?

Reply:
unnecessary